scripts/recipes.js

```diff
@@ -0,0 +1,30 @@
+
+const fs = require('fs');
+const readdir = require('recursive-readdir');
+const yaml = require('js-yaml');
+const { sortBy } = require('lodash');
+
+const allRecipes = [];
+
+const validateRecipes = () => {
+  allRecipes.forEach(item => {
+    if (!item.name) {
```

Comment:
doesnt it also need description, icon, descriptor, interaction, ingredients[].name, descriptor, level?